test/testtask_test.rb

```diff
@@ -3,18 +3,28 @@
 require 'rake/testtask'
 
 class TestTaskTest < Minitest::Test
-  def test_attributes
-    rake_task = nil
+  def setup
+    @name = 'sample_name'
+    @rake_task = nil
     # This will be in users' Rakefiles
-    Rake::TestTask.new { |t|
-      t.libs << "test"
+    @ttnt_task = Rake::TestTask.new { |t|
```

Comment:
Oops, you are right. Thank you for noticing!

Fixed here: https://github.com/Genki-S/ttnt/commit/c660fff10a6020c8d1da7a6b87366874c28b8536